edenai_apis/apis/microsoft/microsoft_llm_api.py

```
@@ -87,3 +87,84 @@ def llm__chat(
             **kwargs,
         )
         return response
+
+    async def llm__achat(
+        self,
+        messages: List = [],
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)